modules/hitlnext/src/views/lang.ts

```diff
@@ -2,9 +2,10 @@ import lang from 'common/lang'
 
 import en from '../translations/en.json'
 import fr from '../translations/fr.json'
+import es from '../translations/es.json'
```

Comment:
Can you please fix the two ESLint warnings? Then I think we would be good to go!